### ISSUE TITLE
kubectl page: Make it more concise 

### ISSUE THREAD

**Issue by xild** [tldr-pages/tldr#3195]
I have a few suggestions to kubectl pages, but I'm not sure if it makes sense for everyone or just for me, so here I go:

use:  
```
 # More information: <https://kubernetes.io/docs/reference/kubectl/cheatsheet/>.
``` 
instead of
```
# More information: <https://kubernetes.io/docs/tasks/tools/install-kubectl>.
``` 

one command: 
```
# List all information about a resource (pods, deployments, services, ingresses, etc.) with more information:
kubectl get pods -o wide
``` 
instead of 2 commands:
```
# List all pods in all namespaces:
kubectl get pods --all-namespaces

# List all pods with more information (such as node name):
kubectl get pods -o wide
``` 

one command: 
```
# Display Resource (CPU/Memory/Storage) usage of nodes or pods
kubectl top node
``` 
instead of two:
```
# Show metrics for all nodes:
kubectl top node

# Show metrics for all pods in the default namespace:
kubectl top pod
```  

and I'd like to see two more commands:
```
# Print the logs for a container in a pod or specified resource.
kubectl logs my-pod 

# Run command in existing pod
kubectl exec my-pod -- ls / 
```   

If it makes sense, I can proceed and submit a pr.

**Comment by schneiderl** [comment]
I'm not aware of the command itself so I won't be commenting on that. 
About the "More information" link, I believe the proposed change makes a lot of sense. 
A cheatsheet of information about how to use the command is way more valuable than a installation link, IMHO. 

About the command itself, lets wait for others opinions.

**Comment by mebeim** [comment]
Hi @xild.

I agree with both you and @schneiderl that the current "more information" link is not that useful, but I would suggest to link to this one instead of the one you just suggested: https://kubernetes.io/docs/reference/kubectl/. It makes more sense to have a link to the documentation which explains how the command works rather than link directly to a cheatsheet (after all, the cheatsheet is still easily reachable by this link too).

About the edits you're suggesting, they seem to make sense to me, and the two additions look fine. Only one note, I would suggest rewording this since it repeats "information" two times:

    List all information about a resource (pods, deployments, services, ingresses, etc.) with more information:

I think you should go ahead and submit a PR for this, I don't see any problem about that, and we could discuss the changes on the PR in more detail.

**Comment by sbrl** [comment]
Hey! Great ideas here. Would you like to open a PR, @xild?

**Comment by xild** [comment]
sure @sbrl, I will work on it tonight.
